_editorial_policies/02_editor_instructions.md

```diff
@@ -0,0 +1,45 @@
+- Makes a final decision after receiving a recommendation from the Associate Editor and reviewers (see below)
+
+### Associate Editor
+
+Before assigning manuscripts for review, editors have several main tasks:
+- Ensure they do not have a conflict of interest with respect to the work they are to analyze; if they do, [dealing with as dictated by editorial policy](https://livecomsjournal.github.io/policies/editorial_board/).
+- Check to ensure that the manuscript has appropropriate style, grammar, layout, and figure quality to be ready for editing, as in the [instructions for authors](https://livecomsjournal.github.io/authors/policies/). Remember, the journal will not be editing the manuscript, so if you will need to reject the manuscript (for additional revision) because of grammar issues or other stylistic reasons, you should do this *before* sending it for review to avoid wasting the time of the reviewers.
+- Identify suitable reviewers, who may include experts suggested by the authors, others in the field you already know of, or authors cited frequently in the article. LiveCoMS generally requires at least two reviewers, though exceptional circumstances (such as extensive community feedback via GitHub) may result in exceptions.
+
+## Review handling
+
+Once an editor has handled the pre-review steps described above, the review process is largely similar to typical journals. The editor:
+- Contacts suitable reviewers to request reviews, giving them a reasonable (typically three weeks) but not extended amount of time to provide their reviews and noting our conflict of interest policy
+- Handles any potential conflict of interests disclosed by reviewers consistent with (editorial policy](https://livecomsjournal.github.io/policies/editorial_board/)
+- Makes reviewers aware of the [review criteria](https://livecomsjournal.github.io/authors/policies/), including category-specific review criteria
+- Performs a check that the GitHub repository is in order, or ensure the reviewers do so.
+- Ensures reviews are submitted and analyzed in a timely manner (all reviews should be submitted in no more than six weeks), reaching out to remind reviewers as needed and solicit additional reviews if reviewers are too slow or their analysis conflicts
+- Potentially helps ensure reviewer feedback is fair
+- Makes a recommendation regarding acceptance, acceptance with minor or major revision, or rejection (keeping in mind the content of the presubmission letter), and communicates this recommendation to the Lead Editor who makes a final decision. Normally the Lead Editor will follow the handling editor's recommendations, but this process is helpful since the Lead will have approved the presubmission letter and thus needs to be involved if a less than favorable decision is to be made.
+- If the decision is that the authors must revise, provides appropriate direction (with the Lead Editor) about which comments to address if needed
```

Comment:
This seems to imply the editor(s) will comb through reviews and cherry-pick to some extent.  I don't foresee this would be desirable except in unusual cases.  How about this ...
"If the decision is that the authors must revise, passes along reviewer comments to authors along with, optionally, additional review comments - i.e., the editor may also act as a reviewer"
If the editor does choose to make a review, does that get treated as a regular review or is it separate?  Your original language seems to suggest it would be separate - i.e., part of cover note returning reviews.   We should clarify.

Reply:
I like your wording. 

Basically, my experience serving as an editor is that sometimes I receive reviews which are clearly wrong or clearly misunderstand some part of the paper. I felt it was my job in such cases to tell the authors that they don't need to address reviewer 2's comment #3 or whatever. I'm (here) basically just try and explain that editors have the leeway to do that, whether or not you want to call this "acting as a reviewer".